package.json

```diff
@@ -29,6 +29,8 @@
     "color-harmony": "^0.3.0",
     "core-js": "^3.1.4",
     "crossfilter": "^1.3.12",
+    "cy": "^0.0.1",
```

Comment:
What's the difference between `cy` and `cy2`? Do we need both?

Reply:
This came automatically after yarn add cy2